back-end/knexfile.js

```diff
@@ -7,10 +7,10 @@
 const path = require("path");
 
 const {
-  DATABASE_URL = "postgresql://postgres@localhost/postgres",
-  DATABASE_URL_DEVELOPMENT = "postgresql://postgres@localhost/postgres",
-  DATABASE_URL_TEST = "postgresql://postgres@localhost/postgres",
-  DATABASE_URL_PREVIEW = "postgresql://postgres@localhost/postgres",
+  DATABASE_URL = "postgresql://postgres:password@localhost/postgres",
+  DATABASE_URL_DEVELOPMENT = "postgresql://postgres:password@localhost/postgres",
+  DATABASE_URL_TEST = "postgresql://postgres:password@localhost/postgres",
+  DATABASE_URL_PREVIEW = "postgresql://postgres:password@localhost/postgres",
   DEBUG,
```

Comment:
Did you know that you never have to change the defaults for environment variables?  Setting the value you want in the `.env` will override the defautlsl